target_postgres/connector.py

```diff
@@ -272,8 +278,13 @@ def pick_individual_type(jsonschema_type: dict):
             return JSONB()
         if "array" in jsonschema_type["type"]:
             return ARRAY(JSONB())
+
+        # string formats
         if jsonschema_type.get("format") == "date-time":
             return TIMESTAMP()
+        if self.interpret_content_encoding:
+            if jsonschema_type.get("contentEncoding") == "base16":
```

Comment:
```suggestion
        if self.interpret_content_encoding and jsonschema_type.get("contentEncoding") == "base16":
```

Reply:
I thought about this one but it's not compatible with the current ruff rules, is it ok to do this with a noqa flag?

Reply:
Yeah, I'd think `ruff format` would split this nicely into multiple lines. It's ok to add `noqa`, though.